hypervisor/init_comm.go

```diff
@@ -64,7 +58,7 @@ func NewVmMessage(m *DecodedMessage) []byte {
 	return msg
 }
 
-func ReadVmMessage(conn *net.UnixConn) (*DecodedMessage, error) {
+func ReadVmMessage(conn *net.UnixConn) (*hyperstartapi.DecodedMessage, error) {
```

Comment:
It seems better if a part of ReadVmMessage() is also moved with a proper name. any suggestion?

`func ReadDecodedMessage(r io.Reader) (*hyperstartapi.DecodedMessage, error)`